e2e/_suites/ingest-manager/features/ingest-manager.feature

```diff
@@ -0,0 +1,49 @@
+@ingest
+Feature: Enable Fleet and Deploy Agent
+
+@enroll
+Scenario: Deploying an agent
+  Given the "Fleet" Kibana setup has been executed
+  When an agent is deployed to Fleet
+  Then the agent is listed in Fleet as online
+    And system package dashboards are listed in Fleet
+    And new documents are inserted into Elasticsearch
+
+@start-agent
+Scenario: Starting the agent starts backend processes
+  When an agent is deployed to Fleet
+  Then filebeat is started
+    And metricbeat is started
+
+@stop-agent
+Scenario: Stopping the agent stops backend processes
+  Given an agent is deployed to Fleet
+  When the agent is stopped on the host
+  Then filebeat is stopped
```

Comment:
we would need probably more like `Then there are '2' metricbeat processes` as we will need to check monitoring and ingesting beats